doc/source/whatsnew/v0.24.0.txt

```diff
@@ -35,7 +35,7 @@ Datetimelike API Changes
 Other API Changes
 ^^^^^^^^^^^^^^^^^
 
--
+- :func:`read_excel` has gained the keyword argument ``usecols_excel`` that receives a string containing comma separated Excel ranges and columns. The ``usecols`` keyword argument at :func:`read_excel` had removed support for a string containing comma separated Excel ranges and columns and for an int indicating the first j columns to be read in a ``DataFrame``. Also, the ``usecols`` keyword argument at :func:`read_excel` had added support for receiving a list of strings containing column labels and a callable. (:issue:`18273`)
```

Comment:
Better written as:

```
- :func:`read_excel:" now has a keyword argument of ``usecols_excel`` which allows you to parse select columns via A1 notation (:issue:`18273`)
```

Reply:
@WillAyd, please, take a look at this: https://github.com/pandas-dev/pandas/pull/20480#discussion_r192605116

@jreback has asked to point what is changing.

@jreback, should I do what @WillAyd is asking or should I keep the message I have already written?